frontend/src/components/CompDataProductDetails.vue

```diff
@@ -3,300 +3,325 @@
     <b-alert variant="warning" v-model="showAlert">
       Computational Data Product is not found!
     </b-alert>
-    <div v-if="item">
-      <b-container fluid>
-        <b-row class="m-4">
-          <b-col cols="6" class="d-flex">
-            <b-card class="d-flex justify-content-center align-items-center w-100 h-100 m-0 p-0">
-              <molecular-structure-img :structure="item.structure" fit-to-content/>
-            </b-card>
-          </b-col>
-
-          <b-col cols="6">
-            <b-row>
-              <b-col cols="8">
-                <b-card>
-                  <div class="dp-row">
-                    <div class="dp-key">Family</div>
-                    <div class="dp-value">{{ item.dye_family }}</div>
-                  </div>
-                  <div class="dp-row">
-                    <div class="dp-key">Names</div>
-                    <div class="dp-value">
-                      <div v-for="(name, index) in item.other_names" :key="index">
-                        {{ name }}
-                      </div>
-                    </div>
-                  </div>
+
+
+    <div class="layout-container">
+      <!-- 左侧导航栏 -->
```

Comment:
@Miu0703, add the comments in English

Reply:
![image](https://github.com/user-attachments/assets/429d8534-1fc5-4fad-9213-cb24ec0ce315)